### PR TITLE
Add Jacoco and improve test coverage

### PR DESCRIPTION
**Pull request by lokiriusz**

### CODE REVIEW

**Comment by lokiriusz** [comment]
Issue #11 is CodeSmell fix.
The other Issues are tests.

File: test/src/com/complexible/common/ClassesTests.java [hunk 1]
```diff
@@ -0,0 +1,113 @@
+package com.complexible.common;
```
Comment (bargergo) [inline test/src/com/complexible/common/ClassesTests.java]: Change package to com.complexible.common.reflect

File: test/src/com/complexible/common/MethodsTest.java [hunk 2]
```diff
@@ -0,0 +1,64 @@
+package com.complexible.common;
```
Comment (bargergo) [inline test/src/com/complexible/common/MethodsTest.java]: Change package to com.complexible.common.reflect

File: test/src/com/complexible/pinto/BeansTests.java [hunk 3]
```diff
@@ -0,0 +1,61 @@
+package com.complexible.pinto;
```
Comment (bargergo) [inline test/src/com/complexible/pinto/BeansTests.java]: Change package to com.complexible.common.beans

File: test/src/com/complexible/pinto/SourceObjectImplTests.java [hunk 4]
```diff
@@ -0,0 +1,30 @@
+package com.complexible.pinto;
```
Comment (bargergo) [inline test/src/com/complexible/pinto/SourceObjectImplTests.java]: Change package to com.complexible.pinto.impl

**Comment by bargergo** [comment]
Looks good to me

**Pull request merged**